src/api.cpp

```diff
@@ -227,7 +233,8 @@ SEXP rcpp_capabilities() {
     SET_STRING_ELT(names, 8, Rf_mkChar("long long"));
     SET_STRING_ELT(names, 9, Rf_mkChar("C++0x unordered maps"));
     SET_STRING_ELT(names, 10, Rf_mkChar("C++0x unordered sets"));
-    SET_STRING_ELT(names, 11, Rf_mkChar("Full C++11 support"));
+    SET_STRING_ELT(names, 11, Rf_mkChar("full C++11 support")); // switched from 'Full' to 'full'
```

Comment:
Purely aesthetics which I usually try not to influence decision making.  All other sentences are lower-case too.

The flag came from the (otherwise completely unused) PR by Christian in 2014 which predates _formal_ C++11 support in R which is now 'a given' so the test was never used anywhere, nor do I expect it to be used.   There is /one/ instance in the package which I changed.